tasks/post_to_database/bin/build.sh

```diff
@@ -123,11 +97,10 @@ zip -qr ../post_to_database.zip .
 let return_code=$?
 cd -
 
-check_rc $return_code "ERROR: Failed to create zip archive."
+check_returncode $return_code "ERROR: Failed to create zip archive."
```

Comment:
Change to established pattern removing `let` or possibly use `run_and_check_returncode`

Reply:
Never mind, leave it as is  didn't see that we need to go back a directory first `cd -` so that we are in the proper place for a cleanup.